R/convert_csv2r.R

```diff
@@ -0,0 +1,58 @@
+#' Convert .csv file to global variable that can be used in merge_everything function.
```

Comment:
Could you add some more documentation and an example? I think you should describe what the .csv file should look like, how it should be structured and what its' purpose is.